llpc/unittests/util/testUtil.cpp

```diff
@@ -86,13 +86,15 @@ TEST(UtilTest, IsStageInMaskAllGraphicsBit) {
 }
 
 TEST(UtilTest, IsStageInMaskMultiple) {
-  const unsigned mask = ShaderStageBit::ShaderStageVertexBit | ShaderStageBit::ShaderStageTessEvalBit |
-                        ShaderStageBit::ShaderStageFragmentBit;
-  EXPECT_TRUE(isShaderStageInMask(ShaderStage::ShaderStageVertex, mask));
-  EXPECT_FALSE(isShaderStageInMask(ShaderStage::ShaderStageTessControl, mask));
-  EXPECT_TRUE(isShaderStageInMask(ShaderStage::ShaderStageTessEval, mask));
-  EXPECT_TRUE(isShaderStageInMask(ShaderStage::ShaderStageFragment, mask));
-  EXPECT_FALSE(isShaderStageInMask(ShaderStage::ShaderStageCompute, mask));
+  {
```

Comment:
Couldn't you put that in a separate test instead and leave the existing ones as-is??

Reply:
Because we have other graphics stage to test, and they should belong to this test group but are mutually exclusive to those traditional stages. So I believe placing them in this test group is more reasonable without adding new groups. This will be too scattered.